src/oedisi/tools/cli_tools.py

```diff
@@ -156,57 +156,97 @@ def validate_optional_inputs(
             component, "container_port"
         ), f"post parameter required for component {component.name} for multi-continer model build"
 
+
+def drop_null_values(model: Any)-> dict:
+    clean_model = {}
+    assert isinstance(model, dict), "input to this function should be a dict"
+    for k, v in model.items():
+        if "_" in k:
+            key_name = k.split("_")
+            if len(key_name[1]) > 2:
+                key_name[1] = key_name[1].capitalize()
+            else:
+                key_name[1] = key_name[1].upper()
+            key_name = "".join(key_name)
+        else:
+            key_name = k    
+                
+        if isinstance(v, dict):
+            clean_model[key_name] = drop_null_values(v)
+        elif isinstance(v, list):
+            new_list = []
+            for val in v:
+                new_list.append(drop_null_values(val))
+            clean_model[key_name] =new_list
```

Comment:
We may want to run ruff to format this to get rid of things like  `=new_list`.

Reply:
ruff weirdly did not catch it 

src\oedisi\tools\cli_tools.py:161:5: D103 Missing docstring in public function
src\oedisi\tools\cli_tools.py:186:5: D103 Missing docstring in public function
src\oedisi\tools\cli_tools.py:208:41: F541 [*] f-string without any placeholders
src\oedisi\tools\cli_tools.py:251:5: D103 Missing docstring in public function
src\oedisi\tools\cli_tools.py:260:5: D103 Missing docstring in public function

fixed it